Resources/Prototypes/Entities/Structures/Walls/girders.yml

```
@@ -61,82 +66,107 @@
   name: reinforced girder
   description: A large structural assembly made out of metal and plasteel; It requires a layer of plasteel before it can be considered a reinforced wall.
   components:
-    - type: Sprite
-      sprite: Structures/Walls/solid.rsi
-      state: reinforced_wall_girder
-    - type: Construction
-      graph: Girder
-      node: reinforcedGirder
-    - type: Destructible
-      thresholds:
-        - trigger:
-            !type:DamageTrigger
-            damage: 200
-          behaviors:
-            - !type:DoActsBehavior
-              acts: [ "Destruction" ]
-            - !type:PlaySoundBehavior
-              sound:
-                collection: MetalBreak
-        - trigger:
-            !type:DamageTrigger
-            damage: 50
-          behaviors:
-            - !type:SpawnEntitiesBehavior
-              spawn:
-                SheetSteel1:
-                  min: 1
-                  max: 1
-                SheetPlasteel1:
-                  min: 1
-                  max: 1
-            - !type:DoActsBehavior
-              acts: [ "Destruction" ]
-            - !type:PlaySoundBehavior
-              sound:
-                collection: MetalBreak
-    - type: StaticPrice
-      price: 130
+  - type: Sprite
+    sprite: Structures/Girders/rgirder.rsi
+  - type: Icon
+    sprite: Structures/Girders/rgirder.rsi
+    state: full
+  - type: IconSmooth
+    key: walls
+    base: rgirder
+  - type: Construction
+    graph: Girder
+    node: reinforcedGirder
+  - type: Destructible
+    thresholds:
+      - trigger:
+          !type:DamageTrigger
+          damage: 200
+        behaviors:
+          - !type:DoActsBehavior
+            acts: [ "Destruction" ]
+          - !type:PlaySoundBehavior
+            sound:
+              collection: MetalBreak
+      - trigger:
+          !type:DamageTrigger
+          damage: 50
+        behaviors:
+          - !type:SpawnEntitiesBehavior
+            spawn:
+              SheetSteel1:
+                min: 1
+                max: 1
+              SheetPlasteel1:
+                min: 1
+                max: 1
+          - !type:DoActsBehavior
+            acts: [ "Destruction" ]
+          - !type:PlaySoundBehavior
+            sound:
+              collection: MetalBreak
```

The indentation here can also be fixed.
```suggestion
    - trigger:
        !type:DamageTrigger
        damage: 200
      behaviors:
      - !type:DoActsBehavior
        acts: [ "Destruction" ]
      - !type:PlaySoundBehavior
        sound:
          collection: MetalBreak
    - trigger:
        !type:DamageTrigger
        damage: 50
      behaviors:
      - !type:SpawnEntitiesBehavior
        spawn:
          SheetSteel1:
            min: 1
            max: 1
          SheetPlasteel1:
            min: 1
            max: 1
      - !type:DoActsBehavior
        acts: [ "Destruction" ]
      - !type:PlaySoundBehavior
        sound:
          collection: MetalBreak
```